src/source/Sctp/Sctp.c

```diff
@@ -1,6 +1,8 @@
 #define LOG_CLASS "SCTP"
 #include "../Include_i.h"
 
+static volatile PSctpSessionControl pSctpSessionControl = NULL;
```

Comment:
We shouldn't use statics here. This can be part of the object graph. Please change the deinitSctpSession to take this object of a parent object that it can use instead of the global

Reply:
This thing out lives peerConnection, so we dont have a right parent object for it.

Reply:
I understand now your point. Let's make this a high-level object that's lifespan will be tracked at the application level.

Have createSctpSession and freeSctpSession. The object created then will be passed to the perr connections?